l2beat/l2beat
# Backend auto reload and tsconfig refinement

Resolves L2B-2133

## What's changed
- Cleaned up `package.json`
- Refined master `tsconfig.json` file - added few declarations
- Added `ts-node-dev` for backend
- Replaced plain `start:dev`(`node -r src server`) with explicit `ts-node-dev` - transpileOnly mode since typecheck is separate thing here

## How to test it?
- Build backend deps
- Run `yarn start:dev`
- Save changes to any files in the dependency tree
- See the reload 🤯
